tests/sentry/api/endpoints/test_organization_api_key_details.py

```diff
@@ -1,70 +1,50 @@
-from django.core.urlresolvers import reverse
-
 from sentry.models import ApiKey
 from sentry.testutils import APITestCase
 
 DEFAULT_SCOPES = ["project:read", "event:read", "team:read", "org:read", "member:read"]
 
 
-class OrganizationApiKeyDetails(APITestCase):
-    def test_api_key_no_exist(self):
-        self.login_as(user=self.user)
-        organization = self.create_organization(name="foo", owner=self.user)
+class OrganizationApiKeyDetailsBase(APITestCase):
+    endpoint = "sentry-api-0-organization-api-key-details"
 
-        path = reverse("sentry-api-0-organization-api-key-details", args=[organization.slug, 2])
+    def setUp(self):
+        super().setUp()
+        self.login_as(self.user)
+        self.api_key = ApiKey.objects.create(
+            organization=self.organization, scope_list=DEFAULT_SCOPES
+        )
 
-        resp = self.client.get(path)
 
-        assert resp.status_code == 404
+class OrganizationApiKeyDetails(OrganizationApiKeyDetailsBase):
+    def test_api_key_no_exist(self):
+        invalid_key = self.api_key.id + 10
```

Comment:
I suppose this is more of a semantics issue, but it seems like it's changing the test from checking that one doesn't exist to checking that we can't find an invalid key.

Reply:
The original line had
```
path = reverse("sentry-api-0-organization-api-key-details", args=[organization.slug, 2])
```
I'm worried that the test is flaky as is because a change to a `setUp` or fixture might accidentally create an API key with an ID of `2`. I think the fullest solution would be to say ```
invalid_key = ApiKey.objects.count() + 1
```
but a quicker fix might just be to set the ID to something really high like 1,000.